resilience4j-retrofit/src/main/java/io/github/resilience4j/retrofit/CircuitBreakerCallAdapter.java

```diff
@@ -63,30 +62,20 @@ private CircuitBreakerCallAdapter(final CircuitBreaker circuitBreaker, final Pre
     }
 
     @Override
-    public CallAdapter<?,?> get(Type returnType, Annotation[] annotations, Retrofit retrofit) {
-        if (getRawType(returnType) != Call.class) {
-            return null;
-        }
+    public CallAdapter<?, ?> get(Type returnType, Annotation[] annotations, Retrofit retrofit) {
+        @SuppressWarnings("unchecked")
+        CallAdapter<Object, Object> nextAdapter = (CallAdapter<Object, Object>) retrofit.nextCallAdapter(this, returnType, annotations);
```

Comment:
AFAICT this is the only way to tell the compiler that what we're going is ok, input welcome though.